src/main/java/org/generationcp/middleware/dao/GermplasmSearchDAO.java

```diff
@@ -193,7 +194,12 @@
 			query.addScalar(GermplasmSearchDAO.LOCATION_NAME);
 
 			for (final String propertyId : germplasmSearchParameter.getAddedColumnsPropertyIds()) {
-				query.addScalar(propertyId);
+				if(propertyId == GermplasmSearchDAO.MALE_PARENT_ID) {
```

Comment:
Use `equals() `method for String comparison. Also please put a comment why we appended '_COL' for MALE_PARENT_ID column. Lastly the code can be simplified to single line using `?` operator.

Reply:
Hi @darla-leafnode , noted. Code updated.